python/python/tests/utils/cluster.py

```diff
@@ -10,7 +10,7 @@
 SCRIPT_FILE = os.path.abspath(f"{__file__}/../../../../../utils/cluster_manager.py")
 
 
-class RedisCluster:
+class ValkeyCluster:
```

Comment:
Before python release, it was decided to leave this as it is

Reply:
Why? I don't really see how changing this to ValkeyCluster is problematic. This is a Valkey client.